shared/actions/devices.js

```diff
@@ -201,12 +176,10 @@ function* deviceSaga(): Saga.SagaGenerator<any, any> {
     ],
     changeWaiting
```

Comment:
Note here (collapsed above by github) we do keep the side effects of those loading calls affecting the waiting. We could handle this in the reducer but I think this is a little cleaner. This way we have reducer cases which deal with one part of the data instead of a bunch of reducer cases touching that same data